### PR TITLE
Update help window with other minor changes

### PR DESCRIPTION
**Pull request by CarrieLi1015**
Updated the help window with more detailed help guideline.

Screenshot 2023-03-30 at 3 37 34 PM

### CODE REVIEW

File: src/main/java/seedu/address/ui/HelpWindow.java [hunk 1]
```diff
@@ -20,6 +20,38 @@ public class HelpWindow extends UiPart<Stage> {
...
+            + "5. deleteTask: Deletes a task for a selected student.\n"
+            + "6. check: Checks the task(s) and score(s) for a selected student.\n"
+            + "7. delete: Deletes a student from the student list.\n"
+            + "8. edit: Edits the student’s information.\n"
+            + "9. find: Finds student(s) from the student list by their name(s).\n"
+            + "10. filter: Filters out the matching result based on tag(s).\n"
+            + "11. markComplete: Marks task as complete.\n"
+            + "12. markInProgress: Marks task as in progress.\n"
+            + "13. markLate: Marks task as late.\n"
+            + "14. exportP: Exports a student's progress as PDF file.\n"
```
Comment (toh-xinyi) [inline src/main/java/seedu/address/ui/HelpWindow.java:39]: The commands can change from camelCase to lowercase

File: src/main/java/seedu/address/ui/HelpWindow.java [hunk 2]
```diff
@@ -20,6 +20,38 @@ public class HelpWindow extends UiPart<Stage> {
...
+            + "Please note that the commands below are used without parameters. "
+            + "These commands will ignore all things that follow behind the command keyword.\n"
+            + "1. list: Lists all students. \n"
+            + "2. switch: Switches between the score list tab and score chart tab.\n"
+            + "3. help: Opens the help window. \n"
+            + "4. clear: Clears all items on the student list.\n"
+            + "5. exit: Closes the application. \n"
+            + "\n"
+            + "Addition: The student’s avatar will change if they have a male or female tag. "
+            + "If a student has no or both the tags, his or her avatar will remain as the default.\n";
```
Comment (toh-xinyi) [inline src/main/java/seedu/address/ui/HelpWindow.java:53]:
```suggestion
            + "Extra info:\n All commands are case insensitive. e.g. `LIST` will be interpreted as `list`. \n"
            + "The student’s avatar will change if they have a male or female tag. "
            + "If a student has no or both the tags, his or her avatar will remain as the default.\n";
```

**Review by toh-xinyi — COMMENTED** [review]
LGTM! Some minor changes to be made 😄

File: src/main/java/seedu/address/ui/HelpWindow.java (hunk 1 above)
Comment (CarrieLi1015) [reply src/main/java/seedu/address/ui/HelpWindow.java:39]: Okie, but may I ask if our UG also need to change to lowercase to keep consistent?

**Comment by CarrieLi1015** [comment]
Thank you so much for your advices!

**Pull request merged**